app/upgrades/v3/upgrades_test.go

```diff
@@ -45,7 +45,8 @@ func (s *UpgradeTestSuite) TestUpgrade() {
 	ctx := s.Ctx
 	keepers := s.App.AppKeepers
 
-	modulesToRemove := []string{"qtransfer", "qoracle", "qvesting"}
+	modulesToRemove := []string{}
+	//modulesToRemove := []string{"qtransfer", "qoracle", "qvesting"}
```

Comment:
If this is intentional, could you remove the commented out part

Reply:
[25a3390](https://github.com/quasar-finance/quasar/pull/859/commits/25a33901594d5d867de775151e2f797be28cb9d2)